crates/lib/src/bootloader.rs

```diff
@@ -87,7 +92,39 @@ pub(crate) fn install_systemd_boot(
     Command::new("bootctl")
         .args(["install", "--esp-path", esp_path.as_str()])
         .log_debug()
-        .run_inherited_with_cmd_context()
+        .run_inherited_with_cmd_context()?;
+
+    match autoenroll {
+        None => return Ok(()),
+        Some(SecurebootKeys { dir, keys }) => {
+            println!("Autoenrolling keys");
+            let path = esp_path.join(SYSTEMD_KEY_DIR);
+            create_dir_all(&path)?;
+
+            let keys_dir = esp_mount
+                .fd
+                .open_dir(SYSTEMD_KEY_DIR)
+                .with_context(|| format!("Opening {path}"))?;
+
+            for filename in keys.iter() {
+                let p = path.join(&filename);
+
+                // create directory if it doesn't already exist
+                if let Some(parent) = p.parent() {
+                    create_dir_all(parent)?;
+                }
+
+                dir.copy(&filename, &keys_dir, &filename)
+                    .with_context(|| format!("Copying secure boot key: {p}"))?;
+                println!("Wrote secure boot key: {p}");
+            }
+            if keys.len() < 1 {
+                println!("No secure boot keys to copy over");
+            }
```

Comment:
Well it's weird to print "Autoenrolling keys". let's consolidate with the above println.